src/components/Home/Home.js

```diff
@@ -57,9 +59,13 @@ class Home extends React.Component {
    */
   title() {
     if (this.props.selectedOrganization) {
-      return `Cluster Overview | ${this.props.selectedOrganization}`;
+      return (
+        'Cluster Overview | ' +
+        this.props.selectedOrganization +
+        ' | Giant Swarm'
```

Comment:
The `| Giant Swarm` suffix gets automatically added by the `useDocumentTitle` hook inside the `DocumentTitle` component